workspaces/liferay-marketplace-workspace/client-extensions/liferay-marketplace-custom-element/src/context/MarketplaceContext.tsx

```diff
@@ -36,6 +47,8 @@ const MarketplaceContextProvider: React.FC<MarketplaceContextProviderProps> = ({
 	children,
 	properties,
 }) => {
+	const [eulaCheckbox, setEulaCheckbox] = useState<boolean>(false);
```

Comment:
I think this shouldn't be here, this is used across the entire Marketplace, but we only use that during the publisher journey, I think it makes sense to add this inside the reducer from:

manage-app-state/AppManageState.tsx
- Add the initial value for eulaCheckbox
manage-app-state/actionTypes.ts
- UPDATE_EULA_CHECKBOX
manage-app-state/reducer.ts
- case UPDATE_EULA_CHECKBOX...